Starting SciPaMaTo-Public with profile development in debug mode throws

Starting SciPaMaTo-Public from IntelliJ with profile `development,public` resulted in the following exception (while starting it without those profiles did not throw):

```
org.apache.wicket.WicketRuntimeException: An error occurred while generating an Url for handler 'ResourceReferenceRequestHandler{resourceReference=scope: ch.difty.scipamato.publ.web.themes.markup.html.publ.ScipamatoPublicLessReference; name: less/__main.less; locale: null; style: null; variation: null, pageParameters=}'
	at org.apache.wicket.request.cycle.RequestCycle.urlFor(RequestCycle.java:557) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.head.CssReferenceHeaderItem.getUrl(CssReferenceHeaderItem.java:162) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.head.CssReferenceHeaderItem.getRenderTokens(CssReferenceHeaderItem.java:149) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.head.internal.HeaderResponse.wasItemRendered(HeaderResponse.java:61) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.head.internal.HeaderResponse.render(HeaderResponse.java:52) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.head.filter.FilteringHeaderResponse.getContent(FilteringHeaderResponse.java:270) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.head.filter.FilteringHeaderResponse.close(FilteringHeaderResponse.java:222) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.html.DecoratingHeaderResponse.close(DecoratingHeaderResponse.java:87) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.head.ResourceAggregator.close(ResourceAggregator.java:290) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.html.internal.HtmlHeaderContainer.onComponentTagBody(HtmlHeaderContainer.java:176) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.html.panel.DefaultMarkupSourcingStrategy.onComponentTagBody(DefaultMarkupSourcingStrategy.java:70) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.Component.internalRenderComponent(Component.java:2484) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.MarkupContainer.onRender(MarkupContainer.java:1593) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.Component.internalRender(Component.java:2289) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.Component.render(Component.java:2220) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.MarkupContainer.renderNext(MarkupContainer.java:1454) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.MarkupContainer.renderAll(MarkupContainer.java:1654) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.Page.onRender(Page.java:858) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.markup.html.WebPage.onRender(WebPage.java:126) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.Component.internalRender(Component.java:2289) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.Component.render(Component.java:2220) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.Page.renderPage(Page.java:998) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.handler.render.WebPageRenderer.renderPage(WebPageRenderer.java:124) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.handler.render.WebPageRenderer.respond(WebPageRenderer.java:195) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.core.request.handler.RenderPageRequestHandler.respond(RenderPageRequestHandler.java:202) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle$HandlerExecutor.respond(RequestCycle.java:914) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.RequestHandlerExecutor.execute(RequestHandlerExecutor.java:65) ~[wicket-request-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.execute(RequestCycle.java:282) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:327) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.executeExceptionRequestHandler(RequestCycle.java:335) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.processRequest(RequestCycle.java:259) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.processRequestAndDetach(RequestCycle.java:221) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.protocol.http.WicketFilter.processRequestCycle(WicketFilter.java:275) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.protocol.http.WicketFilter.processRequest(WicketFilter.java:206) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.protocol.http.WicketFilter.doFilter(WicketFilter.java:299) ~[wicket-core-8.3.0.jar:8.3.0]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at org.springframework.boot.actuate.web.trace.servlet.HttpTraceFilter.doFilterInternal(HttpTraceFilter.java:90) ~[spring-boot-actuator-2.1.3.RELEASE.jar:2.1.3.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:320) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.invoke(FilterSecurityInterceptor.java:127) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.access.intercept.FilterSecurityInterceptor.doFilter(FilterSecurityInterceptor.java:91) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.access.ExceptionTranslationFilter.doFilter(ExceptionTranslationFilter.java:119) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.session.SessionManagementFilter.doFilter(SessionManagementFilter.java:137) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.authentication.AnonymousAuthenticationFilter.doFilter(AnonymousAuthenticationFilter.java:111) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.servletapi.SecurityContextHolderAwareRequestFilter.doFilter(SecurityContextHolderAwareRequestFilter.java:170) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.savedrequest.RequestCacheAwareFilter.doFilter(RequestCacheAwareFilter.java:63) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.authentication.logout.LogoutFilter.doFilter(LogoutFilter.java:116) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.header.HeaderWriterFilter.doFilterInternal(HeaderWriterFilter.java:74) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.context.SecurityContextPersistenceFilter.doFilter(SecurityContextPersistenceFilter.java:105) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.context.request.async.WebAsyncManagerIntegrationFilter.doFilterInternal(WebAsyncManagerIntegrationFilter.java:56) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at org.springframework.security.web.FilterChainProxy$VirtualFilterChain.doFilter(FilterChainProxy.java:334) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy.doFilterInternal(FilterChainProxy.java:215) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.security.web.FilterChainProxy.doFilter(FilterChainProxy.java:178) ~[spring-security-web-5.1.4.RELEASE.jar:5.1.4.RELEASE]
	at org.springframework.web.filter.DelegatingFilterProxy.invokeDelegate(DelegatingFilterProxy.java:357) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at org.springframework.web.filter.DelegatingFilterProxy.doFilter(DelegatingFilterProxy.java:270) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at org.springframework.web.filter.RequestContextFilter.doFilterInternal(RequestContextFilter.java:99) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at org.springframework.web.filter.FormContentFilter.doFilterInternal(FormContentFilter.java:92) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at org.springframework.web.filter.HiddenHttpMethodFilter.doFilterInternal(HiddenHttpMethodFilter.java:93) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.filterAndRecordMetrics(WebMvcMetricsFilter.java:117) ~[spring-boot-actuator-2.1.3.RELEASE.jar:2.1.3.RELEASE]
	at org.springframework.boot.actuate.metrics.web.servlet.WebMvcMetricsFilter.doFilterInternal(WebMvcMetricsFilter.java:106) ~[spring-boot-actuator-2.1.3.RELEASE.jar:2.1.3.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at org.springframework.web.filter.CharacterEncodingFilter.doFilterInternal(CharacterEncodingFilter.java:200) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at org.springframework.web.filter.OncePerRequestFilter.doFilter(OncePerRequestFilter.java:107) ~[spring-web-5.1.5.RELEASE.jar:5.1.5.RELEASE]
	at io.undertow.servlet.core.ManagedFilter.doFilter(ManagedFilter.java:61) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler$FilterChainImpl.doFilter(FilterHandler.java:131) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.FilterHandler.handleRequest(FilterHandler.java:84) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.security.ServletSecurityRoleHandler.handleRequest(ServletSecurityRoleHandler.java:62) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletChain$1.handleRequest(ServletChain.java:68) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletDispatchingHandler.handleRequest(ServletDispatchingHandler.java:36) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.security.SSLInformationAssociationHandler.handleRequest(SSLInformationAssociationHandler.java:132) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.security.ServletAuthenticationCallHandler.handleRequest(ServletAuthenticationCallHandler.java:57) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.security.handlers.AbstractConfidentialityHandler.handleRequest(AbstractConfidentialityHandler.java:46) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.security.ServletConfidentialityConstraintHandler.handleRequest(ServletConfidentialityConstraintHandler.java:64) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.security.handlers.AuthenticationMechanismsHandler.handleRequest(AuthenticationMechanismsHandler.java:60) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.security.CachedAuthenticatedSessionHandler.handleRequest(CachedAuthenticatedSessionHandler.java:77) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.security.handlers.AbstractSecurityContextAssociationHandler.handleRequest(AbstractSecurityContextAssociationHandler.java:43) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.server.handlers.PredicateHandler.handleRequest(PredicateHandler.java:43) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler.handleFirstRequest(ServletInitialHandler.java:292) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler.access$100(ServletInitialHandler.java:81) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:138) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler$2.call(ServletInitialHandler.java:135) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.core.ServletRequestContextThreadSetupAction$1.call(ServletRequestContextThreadSetupAction.java:48) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.core.ContextClassLoaderSetupAction$1.call(ContextClassLoaderSetupAction.java:43) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler.dispatchRequest(ServletInitialHandler.java:272) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler.access$000(ServletInitialHandler.java:81) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.servlet.handlers.ServletInitialHandler$1.handleRequest(ServletInitialHandler.java:104) ~[undertow-servlet-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.server.Connectors.executeRootHandler(Connectors.java:364) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at io.undertow.server.HttpServerExchange$1.run(HttpServerExchange.java:830) ~[undertow-core-2.0.17.Final.jar:2.0.17.Final]
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128) ~[na:na]
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628) ~[na:na]
	at java.base/java.lang.Thread.run(Thread.java:834) ~[na:na]
Caused by: java.lang.IllegalArgumentException: Argument 'lessStream' may not be null.
	at org.apache.wicket.util.lang.Args.notNull(Args.java:41) ~[wicket-util-8.3.0.jar:8.3.0]
	at de.agilecoders.wicket.less.LessResourceStream.<init>(LessResourceStream.java:33) ~[wicket-bootstrap-less-2.0.7.jar:na]
	at de.agilecoders.wicket.less.LessPackageResource.getResourceStream(LessPackageResource.java:31) ~[wicket-bootstrap-less-2.0.7.jar:na]
	at org.apache.wicket.request.resource.caching.version.LastModifiedResourceVersion.getVersion(LastModifiedResourceVersion.java:44) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.resource.caching.version.RequestCycleCachedResourceVersion.getVersion(RequestCycleCachedResourceVersion.java:98) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.resource.caching.FilenameWithVersionResourceCachingStrategy.decorateUrl(FilenameWithVersionResourceCachingStrategy.java:105) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.core.request.mapper.BasicResourceReferenceMapper.mapHandler(BasicResourceReferenceMapper.java:234) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.mapper.ParentPathReferenceRewriter.mapHandler(ParentPathReferenceRewriter.java:92) ~[wicket-request-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.mapper.CompoundRequestMapper.mapHandler(CompoundRequestMapper.java:210) ~[wicket-request-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.mapUrlFor(RequestCycle.java:451) ~[wicket-core-8.3.0.jar:8.3.0]
	at org.apache.wicket.request.cycle.RequestCycle.urlFor(RequestCycle.java:551) ~[wicket-core-8.3.0.jar:8.3.0]
	... 128 common frames omitted
```

--- a/CHANGELOG.asciidoc
+++ b/CHANGELOG.asciidoc
@@ -70,6 +70,7 @@ NOTE: References to user stories are in the form Iteration/Story-Number.
 .Removed
 
 .Fixed
+- {url-issues}73[#73] - Public: Allow starting SciPaMaTo-Public in development profile
 
 .Security
 
@@ -78,7 +79,7 @@ NOTE: References to user stories are in the form Iteration/Story-Number.
 == [1.1.4] -- 2019-03-13
 
 .Added
-- {url-issues}63[#70] - Core: Filter Newsletters by newsletter topic
+- {url-issues}70[#70] - Core: Filter Newsletters by newsletter topic
 - Added many wiki pages
 - Test coverage, refactorings and small optimizations behind the scene
 

--- a/implementation/scipamato/scipamato-public-web/pom.xml
+++ b/implementation/scipamato/scipamato-public-web/pom.xml
@@ -123,6 +123,7 @@
                 <includes>
                     <include>**/*.html</include>
                     <include>**/*.css</include>
+                    <include>**/*.less</include>
                     <include>**/*.properties</include>
                 </includes>
             </resource>

--- a/implementation/scipamato/scipamato-public-web/src/main/java/ch/difty/scipamato/publ/web/themes/markup/html/publ/ScipamatoPublicLessReference.java
+++ b/implementation/scipamato/scipamato-public-web/src/main/java/ch/difty/scipamato/publ/web/themes/markup/html/publ/ScipamatoPublicLessReference.java
@@ -9,12 +9,12 @@ import org.apache.wicket.markup.head.CssHeaderItem;
 import org.apache.wicket.markup.head.HeaderItem;
 
 /**
- * less resource reference that references `scipamato-public-bootstrap.css`. Can
+ * less resource reference that references the main less file. Can
  * be used by the ScipamatoPublicTheme in the development process of the
  * less/css classes. Should not be used in production. Use
  * {@link ScipamatoPublicCssReference} instead.
  * <p>
- * <strong>Usage</strong>
+ * <strong>Usage:</strong>
  *
  * <pre>
  * response.render(CssHeaderItem.forReference(ScipamatoPublicLessReference.instance()));

--- a/implementation/scipamato/scipamato-public-web/src/main/java/ch/difty/scipamato/publ/web/themes/markup/html/publ/ScipamatoPublicTheme.java
+++ b/implementation/scipamato/scipamato-public-web/src/main/java/ch/difty/scipamato/publ/web/themes/markup/html/publ/ScipamatoPublicTheme.java
@@ -8,17 +8,17 @@ import org.apache.wicket.markup.head.CssHeaderItem;
 import org.apache.wicket.markup.head.HeaderItem;
 
 /**
- * java representation of the SciPaMaTo-Public Theme. Is based on TODC Bootstrap
+ * Java representation of the SciPaMaTo-Public Theme. Is based on TODC Bootstrap
  * and the wicket-bootstrap implementation of Michael Haitz in wicket-bootstrap.
  * <p>
  * A flag passed into the constructor indicates whether the precompiled CSS
  * classes will be used (preferable in production) or if the LESS files shall be
  * dynamically compiled into CSS.
  * <p>
- * This theme modifies bootstrap and therefore both files gets loaded
+ * This theme modifies bootstrap and therefore both css files get loaded
  * (`bootstrap.css` and `scipamato-public-bootstrap.css`).
  *
- * <span class="strong">Usage</span>
+ * <span class="strong">Usage:</span>
  *
  * <pre>
  * settings.setThemeProvider(new SingleThemeProvider(new ScipamatoPublicTheme()));
@@ -50,7 +50,7 @@ public class ScipamatoPublicTheme extends Theme {
      * {@literal scipamato-public}.
      *
      * @param useLessOverCss
-     *     if true: SciPamaTo will dynamically compile the LESS files into
+     *     if true: SciPaMaTo will dynamically compile the LESS files into
      *     CSS. If false, the precompiled CSS is used.
      */
     public ScipamatoPublicTheme(final boolean useLessOverCss) {

--- a/implementation/scipamato/scipamato-public-web/src/main/resources/application.properties
+++ b/implementation/scipamato/scipamato-public-web/src/main/resources/application.properties
@@ -34,11 +34,13 @@ scipamato.cms-url-new-study-page=http://localhost:8081/new-studies
 # Commercial font is present
 scipamato.commercial-font-present=false
 
-# Do not show the navbar by default (unless overridden via page paramter showNavbar
+# Do not show the navbar by default (unless overridden via page parameter showNavbar
 scipamato.navbar-visible-by-default=false
 
-# Use the compiled css classes, as opposed to dynamically compile the less sources
+# Use the compiled css classes (false), as opposed to dynamically compiling the less sources (true)
 scipamato.less-used-over-css=false
+
+
 #
 # Web Server Settings
 #

